parser/src/main/java/me/vertretungsplan/parser/LegionBoardParser.java

```diff
@@ -0,0 +1,327 @@
+				throw new IOException("Multi-day changes are not supported yet.");
+			}
+			// If starting date of change does not equal date of SubstitutionScheduleDay
+			if (!change.getString("startingDate").equals(currentDate)) {
+				// If SubstitutionScheduleDay is not empty
+				if (substitutionScheduleDay.getSubstitutions() != null) {
+					substitutionSchedule.addDay(substitutionScheduleDay);
+				}
+				final LocalDate localDate = new LocalDate(change.getString("startingDate"));
+				substitutionScheduleDay.setDate(localDate);
+			}
+			final Substitution substitution = new Substitution();
+			// Set class
+			final String classId = change.getString("course");
+			if (!classId.equals("0") && !classId.equals(null)) {
+				if (coursesHashMap == null) {
+					throw new IOException("Change references a course but courses are empty.");
+				}
+				final String singleClass = coursesHashMap.get(classId);
+				final HashSet<String> classes = new HashSet<>();
+				classes.add(singleClass);
+				substitution.setClasses(classes);
+			}
+			// Set type
+			String type = "Unknown";
+			switch (change.getString("type")) {
+				case "0":
+					type = "Entfall";
+					break;
+				case "1":
+					type = "Vertretung";
+					break;
+				case "2":
+					type = "Information";
+					break;
+			}
+			substitution.setType(type);
+			// Set color
+			substitution.setColor(colorProvider.getColor(type));
+			// Set covering teacher
+			final String coveringTeacherId = change.getString("coveringTeacher");
+			if (!coveringTeacherId.equals("0")) {
+				if (teachersHashMap == null) {
+					throw new IOException("Change references a covering teacher but teachers are empty.");
+				}
+				substitution.setTeacher(teachersHashMap.get(coveringTeacherId));
+			}
+			// Set teacher
+			final String teacherId = change.getString("teacher");
+			if (!teacherId.equals("0")) {
+				if (teachersHashMap == null) {
+					throw new IOException("Change references a teacher but teachers are empty.");
+				}
+				if (type.equals("Vertretung") || !coveringTeacherId.equals("0")) {
+					substitution.setPreviousTeacher(teachersHashMap.get(teacherId));
+				}
+				else {
+					substitution.setTeacher(teachersHashMap.get(teacherId));
+				}
+					
+			}
+			// Set description
+			substitution.setDesc(change.getString("text"));
+			// Set lesson
+			final String startingHour = change.getString("startingHour");
+			final String endingHour = change.getString("endingHour");
+			if (!startingHour.equals("") || !endingHour.equals("")) {
+				String lesson = "";
+				if (!startingHour.equals("") && endingHour.equals("")) {
+					lesson = "Ab " + startingHour + ". Std.";
+				}
+				if (startingHour.equals("") && !endingHour.equals("")) {
+					lesson = "Bis " + endingHour + ". Std.";
+				}
+				if (!startingHour.equals("") && !endingHour.equals("")) {
+					lesson = startingHour + ". - " + endingHour + ". Std.";
+				}
+				substitution.setLesson(lesson);
```

Comment:
The app displays the lesson in quite a large font, so it's probably better to just set the lesson to something like `1` (for single lessons) or `1 - 2` (for multiple lessons).